### PR TITLE
Lower Sound - Changed Keybind

### PR DESCRIPTION
**Pull request by Heavybob**
Using the END key as its simple and does not require a key combination.

### CODE REVIEW

File: Altis_Life.Altis/core/functions/fn_keyHandler.sqf [hunk 1]
```diff
@@ -184,9 +184,9 @@ switch (_code) do {
 		};
 	};
 
-	//O Key
-	case 24: {
-		if(_shift) then {
+	// Lower Sound END Key
+	case 207: {
+		if(playerSide in [civilian,west,independent]) then {
```
Comment (sargken) [inline Altis_Life.Altis/core/functions/fn_keyHandler.sqf]: Why even have the if statement. Its irrelevant.

**Comment by Heavybob** [comment]
Just in case the player isn't assigned a side. *Highly unlikely*

**Comment by sargken** [comment]
Then they wouldn't be in-game, add in opfor though.

**Comment by Heavybob** [comment]
The check does not make sense. Will remove.

**Pull request closed without merging**